pkg/metrics/metrics.go

```diff
@@ -89,9 +89,25 @@ var (
 			NodePoolLabel,
 		},
 	)
+	IgnoredPodCount = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Namespace: Namespace,
+			Name:      "ignored_pod_count",
+			Help:      "Number of pods ignored during scheduling by Karpenter",
+		},
+	)
+	NodesEvictionRequestsTotal = prometheus.NewCounterVec(
```

Comment:
This is only relevant to termination -- should this be in that package's metrics file